lua/astrocommunity/completion/avante-nvim/init.lua

```diff
@@ -4,16 +4,6 @@ return {
   cmd = {
     "AvanteAsk",
     "AvanteBuild",
-    "AvanteConflictChooseAllTheirs",
-    "AvanteConflictChooseBase",
-    "AvanteConflictChooseBoth",
-    "AvanteConflictChooseCursor",
-    "AvanteConflictChooseNone",
-    "AvanteConflictChooseOurs",
-    "AvanteConflictChooseTheirs",
-    "AvanteConflictListQf",
-    "AvanteConflictNextConflict",
-    "AvanteConflictPrevConflict",
     "AvanteEdit",
     "AvanteRefresh",
     "AvanteSwitchProvider",
```

Comment:
```suggestion
    "AvanteSwitchProvider",
    "AvanteChat",
    "AvanteToggle",
    "AvanteClear",
```

I think with this we should also add in some missing commands as well!